src/app/data/key-map.js

```diff
@@ -0,0 +1,104 @@
+const keyMap = [
```

Comment:
It is better to create two separate maps for EN/RU in case not to use magic indexes `0`/`1` for `keyVal`.